ingestion/src/metadata/ingestion/source/metadata/atlas.py

```diff
@@ -126,20 +124,38 @@ def prepare(self):
         """
 
     def next_record(self):
-        for key in ENTITY_TYPES["Table"]:
-            self.service = self.get_database_service()
-            self.tables[key] = self.atlas_client.list_entities(entity_type=key)
+        for service in self.service_connection.databaseServiceName or []:
 
-        for key in ENTITY_TYPES.get("Topic", []):
-            self.message_service = self.get_message_service()
-            self.topics[key] = self.atlas_client.list_entities(entity_type=key)
+            try:
+                create_service = self.metadata.get_by_name(
+                    entity_type=DatabaseService, fqn_search_string=service
+                )
+                for key in ENTITY_TYPES["Table"]:
```

Comment:
is it expected for `ENTITY_TYPES` to have `table` key twice?

Reply:
yes